### PR TITLE
Add simple performance test for document v1

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review briefly. Draft for now — I would like to verify locally as well :)

### CODE REVIEW

**Comment by jonmv** [comment]
Works and ready for review and merge. PTAL @bjorncs or @geirst  :)

**Comment by jonmv** [comment]
Hmm, this is weird, though: all documents should be present, but this is the breakdown for the GET part. POST part has all 200 OKs and more than 65k successes ... 
```
[14:25:51.311]        200 :   326144
[14:25:51.311]        404 :    64458
```

**Comment by jonmv** [comment]
Aha, looks like an `fbench` artefact: the ratio is even worse for the old handler, which manages only half as many PUTs — apparently, the fbench clients aren't cooperatively looping through the data multiple times.

File: tests/performance/document_rest_api/document_v1_throughput.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    start
+    benchmark_operations(@test_config)
+  end
+
+  def benchmark_operations(methods)
+    qrserver = @vespa.container["combinedcontainer/0"]
+    methods.each do |method, config|
+      paths_file = dirs.tmpdir + method + ".txt"
+      qrserver.execute("for i in {1..#{1 << 16}};"\
```
Comment (bjorncs) [inline tests/performance/document_rest_api/document_v1_throughput.rb]: Consider writing converting this inline shell script to Ruby.

**Review by bjorncs — APPROVED** [review]

File: tests/performance/document_rest_api/document_v1_throughput.rb (hunk 1 above)
Comment (jonmv) [reply tests/performance/document_rest_api/document_v1_throughput.rb]: Yes, will consider. This is still WIP :)

**Pull request merged**